properties.py

```diff
@@ -0,0 +1,79 @@
+import json
+import jsonschema
+import gi
+gi.require_version('Gst', '1.0')
+from gi.repository import GObject, Gst, GLib
+
+
+def load_from_json(pipeline, json_file='props.json', schema_file='props_schema.json'):
+    with open(json_file, 'r') as f:
+        properties = json.load(f)
+    with open(schema_file, 'r') as f:
+        schema = json.load(f)
+
+    try:
+        jsonschema.validate(properties, schema)
+    except jsonschema.exceptions.ValidationError as e:
+        print('not matching to schema', e)
+        return
+
+    for element in pipeline.children:
+        for prop in element.props:
+            try:
+                json_config = properties[element.name][prop.name]
+            except KeyError:
+                continue
+            setattr(element.props, prop.name, json_config)
+
+def dump_to_json(pipeline, json_file='props.json'):
+    elements = {}
+    for element in reversed(pipeline.children):
+        current_element = {}
+        for prop in element.props:
+            if 'Gst' in prop.value_type.name:
+                continue
+            current_element[prop.name] =  [prop.flags, 
```

Comment:
is the way `props.flags` is represented will work with the JsonScheme? does JsonScheme accepts RO/RW as an attribute (I really don't know :))

Reply:
Using 'const' for RO attributes